static/app/utils/discover/fields.tsx

```diff
@@ -118,6 +118,18 @@ export type Column = QueryFieldValue;
 
 export type Alignments = 'left' | 'right';
 
+export enum RateUnits {
+  PER_SECOND = '1/second',
+  PER_MINUTE = '1/minute',
+  PER_HOUR = '1/hour',
+}
+
+export const RATE_UNIT_LABELS = {
+  [RateUnits.PER_SECOND]: '/s',
+  [RateUnits.PER_MINUTE]: '/min',
+  [RateUnits.PER_HOUR]: '/hr',
```

Comment:
l: As per the [non-SI units](https://en.wikipedia.org/wiki/Non-SI_units_mentioned_in_the_SI) still considered SI, hour is typically represented by `h`.

Do we want to switch to `/h` then?

Reply:
Sold!